sdk/client/src/verifying_client.rs

```diff
@@ -0,0 +1,516 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    client::Client,
+    request::MethodRequest,
+    response::{MethodResponse, Response},
+    state::State,
+};
+use anyhow::{bail, ensure, format_err, Result};
```

Comment:
We shouldn't use anyhow since this is a library. For some reason its listed as a dependency but its not currently used in this crate.

Reply:
Definitely. I was mostly trying to get things working first' I'll make a pass filling out the real error types : )

Reply:
added real error types!